kernelci.org/content/en/kernel-community/_index.md

```diff
@@ -10,8 +10,8 @@ The Kernel Community is the audience of KernelCI. Upstream maintainers and devel
 > This documentation is still a **WIP**. As we ready new KernelCI for more users the documentation is being improved and expanded.
 
 This section of the documentation shares instructions to:
-* [enable kernel testing for your tree/branch](../api_pipeline/pipeline/developer-documentation/#enabling-a-new-kernel-tree)
-* [enable specific tests](../api_pipeline/pipeline/developer-documentation/#enabling-a-new-test)
+* [enable kernel testing for your tree/branch](../maestro/pipeline/developer-documentation/#enabling-a-new-kernel-tree)
+* [enable specific tests](../maestro/pipeline/developer-documentation/#enabling-a-new-test)
```

Comment:
These links are 404 for me

Reply:
These are working for me. Can you re-check? Usually I need to restart hugo when folders move around.

Reply:
Maybe I'm using different git submodule versions to you. When I'm using the `new-architecture` branch I get:
```
$ git submodule status
 4ef03b1a9e2e73528a9f75b09dcc54ef00a964bc external/kcidb (staging-20240429.0-3-g4ef03b1)
 e6d8197a217a724048260c9d15543fd1d6f0ddfe external/kernelci-api (kernelci-20240527.0)
 21e083027d5ce12bd4cc9b43f9d6bbfa880be996 external/kernelci-core (kernelci-20240527.0)
 1c8a8c216c1b469490e50e97332a1ef205930dd1 external/kernelci-pipeline (staging-20240702.1~83)
 efc76de3ced819f3844243a81f8b93f290c6a7f1 themes/docsy (v0.1.0~146)
```

If I switch to using the `main` branch for `external/kernelci-pipeline` then I no longer get the 404s.

Reply:
The links are working in [staging](https://static.staging.kernelci.org/kernel-community/), so I guess it's an env issue my side.